docs/data/base/guides/interoperability-with-tailwind-css/interoperability-with-tailwind-css.md

```diff
@@ -0,0 +1,451 @@
+# Interoperability with Tailwind CSS
```

Comment:
```suggestion
# Using Tailwind CSS
```
I figure that technically "_interoperability_" might be what happens (two (not necessarily designed to work together) libraries working together?) but this word is terribly confusing and complex to me 😅 Guess we could go with something simpler, what y'all think?

Reply:
I will leave the decision to @samuelsycamore :D I will update the files + pages name together based on what we decide to go with :)

Reply:
+1 for replacing "Interoperability" here and everywhere else in our docs! In this case I think I would lean toward "Working with" but that's very compared to "Using."

```suggestion
# Working with Tailwind CSS
```